src/playground/service.clj

```diff
@@ -81,7 +81,7 @@
     ["/invoices-update/:id" :post (into common-interceptors [http/json-body (param-spec-interceptor ::invoices.update/api :form-params) `invoices.update/perform])]
     ["/invoices/:id" :get (conj common-interceptors (param-spec-interceptor ::invoices.retrieve/api :path-params) `invoices.retrieve/perform)]
     ["/invoices" :get (conj common-interceptors `invoices.retrieve-all/perform)]
-    ["/invoices-delete" :get (into component-interceptors [http/json-body `invoices.delete/perform])]})
+    ["/invoices-delete/:id" :get (into common-interceptors [http/json-body (param-spec-interceptor ::invoices.delete/api :path-params) `invoices.delete/perform])]})
```

Comment:
should this be a POST now?

Reply:
Ideally it would be a DELETE.

You can disregard that fact though, as it introduces some complexity: forms and links don't support DELETE, so you'd have to use AJAX which does support it.

But it's a relatively minor problem